data.sql

```diff
@@ -6,3 +6,13 @@ INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUE
 ('Pikachu','2021-01-07',1,false,15.04),
 ('Devimon','2017-05-12',5,true,11);
 
+/* new insert statements */
+
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg, species) VALUES
+('Charmander', '2020-02-08', 0, false, -11.00, 'Fire Lizard'),
+('Plantmon', '2021-11-15', 2, true, -5.7, 'Plant Creature'),
+('Squirtle', '1993-04-02', 3, false, -12.13, 'Water Turtle'),
+('Angemon', '2005-06-12', 1, true, -45.00, 'Celestial'),
+('Boarmon', '2005-06-07', 7, true, 20.4, 'Boar'),
+('Blossom', '1998-10-13', 3, true, 17.00, 'Plant Blossom'),
+('Ditto', '2022-05-14', 4, true, 22.00, 'Transforming Blob');
```

Comment:
- [ ] Looks like you added entries in `species` column while they haven't been given yet, as with steps move you know there was step where you have to update species column for each animal entry. I would suggest that you correct on it. To be specific, I'd suggest to revert back so that you add entries in the `animals` table without specifying the species since it hasn't been given yet. By doing this, you will be able to follow correctly in sync with the requirements given

Reply:
Oh. thanks for pointing it out, it has been corrected,